src/App.tsx

```diff
@@ -1,57 +1,34 @@
-import React from 'react';
+import React, { useMemo, useState } from 'react';
+
 import './App.scss';
+import { Dropdown } from './components';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [onSelected, setOnSelected] = useState<Person | null>(null);
+
+  const selectedPerson = useMemo(() => {
+    if (onSelected) {
+      const { name, born, died } = onSelected;
+
+      return `${name} (${born} - ${died})`;
+    }
+
+    return 'No selected person';
+  }, [onSelected]);
```

Comment:
You don't have to memoize this value cause you don't pass it as props, instead, you can create a variable for the title text, e.g.:
```js
const titleText = selectedPerson
    ? `${selectedPerson.name} (${selectedPerson.born} = ${selectedPerson.died})`
    : 'No selected person';